model.js

```diff
@@ -10,10 +10,16 @@ function returnTopics(){
 }
 
 function returnArticle(){
-    return db.query("SELECT * FROM articles WHERE article_id = $1" ,[article_id]) // do I need RETURNING?
+    return db.query("SELECT * FROM articles WHERE article_id = $1" ,[article_id]) 
     .then(data => {
         console.log (data);
     })
 }
 
-module.exports = {returnTopics,returnArticle}
\ No newline at end of file
+function returnArticles(){
+    return db.query("SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes,articles.article_img_url,COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id,articles.title,articles.topic,articles.author, articles.created_at,articles.votes,articles.article_img_url ORDER BY articles.created_at DESC; ")
```

Comment:
You shouldn't have to GROUP BY that many things. You should get away with just grouping by comments.article_id, please try and amend this

Reply:
Hi Poonam
I'm not sure why but my test fails when I try to group it by comments.article_id
I can't remember exactly what I did originally as it was late but I had to try a lot of combinations of GROUP BY to get it back how it needed to be.

Reply:
actually its ok. I got it working with articles.article_id